source/agora/consensus/ValidatorSet.d

```diff
@@ -34,6 +34,8 @@ import d2sqlite3.library;
 import d2sqlite3.results;
 import d2sqlite3.sqlite3;
 
+import ocean.text.convert.Formatter;
```

Comment:
Can we just fix the bindings instead ?

Reply:
Alright. I'll remove the last commit for now.